### PR TITLE
Making tests run locally with auth guards off

### PR DESCRIPTION
**Pull request by stullifugl**
# \<Description\>

Adding tests for auth-api

## What

Setting up the test suite for auth-api project

## Why

So that unit tests can be run on controller functions based on the migrate and seed files entries

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/infra-nest-server/src/lib/testServer.ts
Comment (darrikonn) [inline libs/infra-nest-server/src/lib/testServer.ts:1]: Remove commented import.

Btw. this is the only file that I should review.
If you take a look at the CODEOWNERS file, fuglar codeowners has been commented out for the code above, do you know the reason for that?

Otherwise this is approved from me, but I can't set approve due to the `automerge label` + `missing codeowner settings`.

**Review by valurefugl — APPROVED** [review]

**Review by darrikonn — APPROVED** [review]

File: libs/infra-nest-server/src/lib/testServer.ts
Comment (stullifugl) [reply libs/infra-nest-server/src/lib/testServer.ts]: The import is now removed. 

I only wanted you to check out this file since I am adding a new function that overrides the guards for the testing environment. My teammates in Fuglar will review the rest 👍

**Pull request merged**
